presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -471,6 +482,7 @@ public void testInsert()
     public void testDelete()
     {
         featuresToTest(DELETE);
+
```

Comment:
This adds just an empty line here, should be part of earlier commit.

btw, with annotation based approach, there is no guessing whether there should be an empty line before tested-features declaration and the rest of the test body